addons/product/i18n/product.pot

```diff
@@ -2548,6 +2548,12 @@ msgid ""
 "%s"
 msgstr ""
 
+#. module: product
+#: code:addons/product/models/product_pricelist.py:0
+#, python-format
+msgid "You cannot delete those pricelist(s):\n(%s)\n, they are used in other pricelist(s):\n%s"
```

Comment:
@mart-e I've the feeling that every time there is a \n in the translated string, it's split into multiple pot lines but I don't know if there is a documentation on how to write correct .pot files (or the command to run?) :thinking:.

Reply:
Not sure, I wonder if it's a difference of version of `polib` that handles these differently. Just export the .pot from the settings and it should be fine.

Reply:
I didn't know the existence of the translations export wizard, I always did it manually :D.
Well, pot exported and fixing missing translations from #67730 :).

Reply:
You write pot entires manually ? :scream:

Reply:
> I didn't know the existence of the translations export wizard, I always did it manually :D.
> Well, pot exported and fixing missing translations from #67730 :).

Clearly linked to the lack of fromage eating.

Reply:
> You write pot entires manually ?

Until now, for new translated strings in stable, yes :D.